ethers-providers/src/lib.rs

```diff
@@ -561,3 +561,17 @@ pub trait Middleware: Sync + Send + Debug {
             .map_err(FromErr::from)
     }
 }
+
+#[cfg(feature = "celo")]
+#[async_trait]
+pub trait CeloMiddleware: Middleware {
+    async fn get_validators_bls_public_keys(
+        &self,
+        block_number: String,
+    ) -> Result<Vec<String>, ProviderError> {
```

Comment:
How long are the keys? Is there any more specific type we could use instead of String?

Reply:
The public keys are 96 bytes. Maybe we can use `Vec<u8>` instead.